src/components/keypad.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] Please do not disable linters, they are there to help you. Remember that writing a good code is very beneficial.